src/Resources/contao/dca/tl_layout.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the ContaoSwiper Bundle.
+ *
+ * (c) Fritz Michael Gschwantner <https://github.com/fritzmg>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ContaoSwiperBundle\Resources\contao\dca;
+
+$table = 'tl_layout';
+
+$GLOBALS['TL_DCA'][$table]['palettes']['default'] = str_replace('{script_legend}', '{script_legend},add_swiper_scripts', $GLOBALS['TL_DCA'][$table]['palettes']['default']);
```

Comment:
It would be better to use the `PaletteManipulator` instead :)
```php
\Contao\CoreBundle\DataContainer\PaletteManipulator::create()
    ->addField('add_swiper_scripts', 'script_legend', \Contao\CoreBundle\DataContainer\PaletteManipulator::POSITION_APPEND)
    ->applyToPalette('default', 'tl_layout')
;
```

Reply:
This is very handy. I did not know that there is this class for manipulating the palette :+1:  :)
But I had to use the `scripts`-field and set the position to `POSITION_BEFORE` because somehow it did not work with the legend.